apps/backend/src/main/java/ch/heig/pdg/backend/exception/exceptions/UnauthorizedException.java

```diff
@@ -1,6 +1,9 @@
 package ch.heig.pdg.backend.exception.exceptions;
 
 public class UnauthorizedException extends RuntimeException {
+    public UnauthorizedException() {
```

Comment:
I understand the idea of being willing to use `orElseThrow`. But wouldn't it make sense to at least notify why the 401 happened ? I.e. being able to separate between unexisting username and wrong password ?

Reply:
I do not agree a failed login must be obfuscated if it's the password or the username the problem.